### PR TITLE
declaring local variables where needed instead of at the top

### PR DESCRIPTION
**Pull request by Broxzier**
This PR cleans up the code a bit by declaring variables when they are first assigned, sometimes in a different scope when not needed anywhere else. By hand this is going to take a while, so I'm dropping it now. Perhaps clang-tidy has a module for this. Perhaps @janisozaur knows more.

### CODE REVIEW

File: src/cmdline/CommandLine.cpp [hunk 1]
```diff
@@ -159,16 +158,15 @@ namespace CommandLine
...
         // Print usage
         const char * usageString = "usage: openrct2 ";
         const size_t usageStringLength = String::LengthOf(usageString);
         Console::Write(usageString);
 
         // Get the largest command name length and parameter length
         size_t maxNameLength = 0;
-        size_t maxParamsLength = 0;
+		size_t maxParamsLength = 0;
+		const CommandLineCommand * command;
```
Comment (IntelOrca) [inline src/cmdline/CommandLine.cpp]: Use spaces only in C++ files.

Comment (Broxzier) [inline src/cmdline/CommandLine.cpp]: Just changed it :+1:

**Comment by IntelOrca** [comment]
I wouldn't spend too much time doing this as most of the C will change eventually when we refactor and clean up.

**Comment by janisozaur** [comment]
I doubt clang-tidy would reorder variables.

**Pull request merged**